app/views/fake/index.html.erb

```diff
@@ -13,20 +13,74 @@
 
     <div class="col-lg-8">
       <div class="card shadow">
+        <div class="card-header">
+          <h5 class="card-title">Look a card</h5>
+          <a href="#" class="btn btn-primary">Action <span class="material-icons" aria-label="hidden">keyboard_arrow_right</span></a>
+        </div>
```

Comment:
Styles are now applied in the cards.scss file. Can still use light bootstrap classes for alignment only when needed.